lib/assessment.md

```diff
@@ -138,7 +174,11 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will be located on the right side of the screen.
+divTwo
+divThree
+divFour will be located at bottom left
+divFive
```

Comment:
You're right about divOne, but I'm not clear on what you're indicating for the others; unfortunately, I can't give you credit for that.

Try copying the HTML and CSS here into CodePen and seeing what happens - that might help buildup your intuition on this.